src/components/Field/Field.tsx

```diff
@@ -70,14 +74,26 @@ export class Field extends React.PureComponent<FieldProps> {
     return (
       <div className={classes}>
         {label ? <Header sub>{label}</Header> : null}
-        <Input
-          value={value}
-          type={isAddress ? 'text' : type}
-          icon={icon}
-          loading={loading && !isAddress}
-          disabled={disabled}
-          {...rest}
-        />
+        {type === DATE_TYPE ? (
```

Comment:
Would you mind checking if there's a field type enum that comes from the typescript types that we can use here?